packages/calcite-components/src/components/autocomplete-item/autocomplete-item.scss

```diff
@@ -61,6 +61,12 @@
   color: var(--calcite-autocomplete-heading-text-color, var(--calcite-color-text-1));
 }
 
+.text-match {
```

Comment:
Since you're adding a reusable util, can we move this into a mixin?

Reply:
The mixin doesn't make sense at this time since the styles between autocomplete and combobox are not consistent. Hopefully #11173 can resolve that.

Reply:
Sounds good. I'll add a note about this to the issue.